lib/config_module/config_helper.rb

```diff
@@ -41,15 +44,15 @@ def load_namespaces_from tree
           )
         end
       end
-    rescue TypeError => error
```

Comment:
Can we keep this error in the InvalidNamespaceError so if something else happens lower in the call chain we don't obscure the original error?

Reply:
In newer Ruby versions, this error will be in an attribute of the newly raised exception.

Reply:
Will that happen automatically even if we instantiate a new error manually?

Reply:
Yes, in Ruby 2.1+.